deployments/container/cleanup/cleanup.sh

```diff
@@ -52,6 +52,10 @@ microdnf remove -y \
 microdnf remove -y \
     $(rpm -qa | sort | grep -v -f package-names.minimal -e gpg-pubkey)
 
+# We need mount for our init container and install it here.
+# TODO: We need to provide a better way to install this / skip the cleanup of this package.
+microdnf install --assumeyes util-linux-core
```

Comment:
I don't think we want to be pulling anything dynamically from the internet here. People can (and do) try to run this component in an airgapped environment.

Reply:
the following are already in the `minimal` image:
```
$ cat package-list.minimal | grep openssl
openssl-fips-provider-3.0.7-2.el9.x86_64
openssl-libs-3.0.7-28.el9_4.x86_64
```

Reply:
This is the full diff between the packages in the `ubi9-minimal` image and the one we construct here:
```
$ diff package-list.minimal package-list.cleaned
19,21c19,21
< glibc-2.34-100.el9_4.3.x86_64
< glibc-common-2.34-100.el9_4.3.x86_64
< glibc-minimal-langpack-2.34-100.el9_4.3.x86_64
---
> glibc-2.34-100.el9_4.4.x86_64
> glibc-common-2.34-100.el9_4.4.x86_64
> glibc-minimal-langpack-2.34-100.el9_4.4.x86_64
46d45
< libdnf-0.69.0-8.el9_4.1.x86_64
57d55
< libpeas-1.30.0-4.el9.x86_64
70d67
< libusbx-1.0.26-1.el9.x86_64
79d75
< microdnf-3.9.1-3.el9.x86_64
104a101
> util-linux-core-2.37.4-18.el9.x86_64
```
so a slightly newer glibc packge version (would have to check the reason for this) and the addition of `util-linux-core`. Everything else is either removed or the same.

Reply:
OK, I'm fine with this change if you are then.

Reply:
It took me way longer than I'd like to admit to understand what `assumeyes` meant. I kept reading it as "assume eyes". Why this flag wasn't named `--assume-yes` instead is beyond me